src/modules/TeamsList/index.tsx

```diff
@@ -21,22 +22,22 @@ export const TeamsList: FC = () => {
   if (error) return <Error />;
 
   return (
-    <div>
-      <p>Teams length {teams.count}</p>
-      {teams &&
-        teams.results.map((item: Team) => {
-          return <div key={item.id}>Team №{item.number}</div>;
-        })}
-      <p>This is teams list!</p>
+    <StyledTeams>
+      {/* TODO: Replace when getting real data "myTeam" */}
+      <Teams teams={teams} myTeam={teams.results[0]} />
+
       {userData && <p>My github: {userData.github}</p>}
-      {userData && <p>My firstName: {userData.firstName}</p>}
-      {userData && <p>My lastName: {userData.lastName}</p>}
-      {userData && <p>My telegram: {userData.telegram}</p>}
-      {userData && <p>My discord: {userData.discord}</p>}
-      {userData && <p>My score: {userData.score}</p>}
-      {userData && <p>My country: {userData.country}</p>}
-      {userData && <p>My city: {userData.city}</p>}
-      {userData && <p>My courseIds: {userData.courseIds[0]}</p>}
-    </div>
+      {/*{userData && <p>My firstName: {userData.firstName}</p>}*/}
```

Comment:
remove this fill data from file. It`s unused.

Reply:
done